gallery/src/styles/NxList/NxListClickableExample.tsx

```diff
@@ -22,7 +22,7 @@ const NxListClickableExample = () =>
       <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
     </li>
     <li className="nx-list__item selected" tabIndex={0}>
-      <span className="nx-list__text">This list item is selected</span>
+      <span className="nx-list__text" aria-selected="true">This list item is selected</span>
```

Comment:
PR description says you didn't add this?

Reply:
I'd recommend putting aria-selected on the element with the tabIndex set to ensure that screen readers will stop on that element to read it.

Reply:
> PR description says you didn't add this?

I tested by adding it in the span and li, I missed deleting it from the span.

Reply:
@rpokorny 34ddd91819afb9e23d3790004adc1354165df210
@Blacktiger the aria-selected doesn't seem necessary for the screen reader to stop and read it, and it does not make the screen reader say anything about it being selected.